source/includes/_email_stats.md

```diff
@@ -0,0 +1,91 @@
+#Email Stats
+
+```sh
+curl -H "Authorization: Bearer myaccesstoken" "https://api.wootric.com/v2/email_stats"
+
+{
+  "email_stats": {
+    "send_date": "Aug  3, 2023-Sep  3, 2024",
+    "send_requests": 7,
+    "emails_sent": 5,
+    "delivered": 5,
+    "delivered_percentage": "100.0%",
+    "bounce_rate": 0,
+    "bounce_rate_percentage": "0.0%",
+    "open_rate": "60%",
+    "response_rate": "0%",
+    "not_eligible": "2 (29%)",
+    "raw_results": {
+      "processed": 5,
+      "bounce": 0,
+      "dropped": 0,
+      "delivered": 5,
+      "opted_out": 2,
+      "not_delivered": 0,
+      "open": 3,
+      "spamreport": 0,
+      "not_eligible": 0,
+      "total_surveys": 5,
+      "total_responses": 0,
+      "sart_date": "2023-08-03T00:00:00-07:00",
+      "end_date": "2024-09-03T23:59:59-07:00"
+    }
+  },
+  "initial_send": null,
+  "reminder_send": null
+}
+```
+
+Returns email stats information depending on the date_range selected.
+
+Param | Type | Description
+----- | ---- | ------------
+date_range | Hash | See **date_range** parameters below
+
+## Sources
+
+```sh
+curl -H "Authorization: Bearer myaccesstoken" "https://api.wootric.com/v2/email_stats/sources"
+
+
+{
+  "ongoing": [
+    {
+      "id": "email_api",
+      "name": "Email API *"
+    },
+    {
+      "id": "webhook_trigger",
+      "name": "Webhook Trigger *"
+    }
+  ],
+  "sends": []
```

Comment:
Maybe you can put an example with sends, showing the expected structure, this is from one of my accounts

```
{
  "ongoing": [
    {
      "id": "email_api",
      "name": "Email API *"
    },
    {
      "id": "webhook_trigger",
      "name": "Webhook Trigger *"
    }
  ],
  "sends": [
    {
      "id": 29573,
      "name": "Q4_file.csv",
      "date": "Nov 16, 2022  2:18AM"
    },
    {
      "id": 29574,
      "name": "Raw BAU Q3.csv",
      "date": "Nov 16, 2022  2:35AM"
    },
    {
      "id": 29575,
      "name": "Raw BAU Q4.csv",
      "date": "Nov 16, 2022  2:35AM"
    }
  ]
}
```

Reply:
@pmonfort thanks for the comment! I added the example 👍